packages/html/ds/src/breadcrumbs/breadcrumbs.html

```diff
@@ -0,0 +1,30 @@
+<nav aria-label="Breadcrumbs" class="gi-breadcrumbs">
```

Comment:
Could you add a macro to create this structure?
I think we can accept as a param a list of items like this:
```
{
   "label": string;
   "href": string;
   "ellipsis": boolean;
}
```
So this example will be achieved using:
```
{{
   govieBreadcrumbs([
      {
         "label": "Home",
         "href": "/"
      },
      {
         "ellipsis": true
      },
      {
         "label": "Travel",
         "href": "/travel"
      },
      {
         "label": "Documentation",
         "href": "/travel/docs"
      }
   ])
}}
```